📝 Docs: Suggested `pnpm run dev` command doesn't exist in root or packages?

When I tried going through this part of the contributing guide:

https://github.com/aidenybai/million/blob/d8b5c212ea41e1f811a25376a75ae224515d9324/.github/CONTRIBUTING.md#L112-L117

Running `pnpm run dev` the root directory gives:

```plaintext
joshgoldberg ~/repos/million $ pnpm run dev
 ERR_PNPM_NO_SCRIPT  Missing script: dev
```

Same result for the `packages/` directory:

```plaintext
joshgoldberg ~/repos/million/packages $ pnpm run dev
 ERR_PNPM_NO_SCRIPT  Missing script: dev
```

Should `dev` be replaced with `build` or some other command? Note that `pnpm run dev` _does_ exist in `website/package.json` - just not `package.json`. 

Also, I don't think it's clear from the docs whether that command should be run in the root directory or `packages/`. It'd be nice to clarify.

Great contributing guide otherwise btw! 🙌

> When I tried going through this part of the contributing guide:
> 
> https://github.com/aidenybai/million/blob/d8b5c212ea41e1f811a25376a75ae224515d9324/.github/CONTRIBUTING.md#L112-L117
> 
> Running `pnpm run dev` the root directory gives:
> 
> ```
> joshgoldberg ~/repos/million $ pnpm run dev
>  ERR_PNPM_NO_SCRIPT  Missing script: dev
> ```
> 
> Same result for the `packages/` directory:
> 
> ```
> joshgoldberg ~/repos/million/packages $ pnpm run dev
>  ERR_PNPM_NO_SCRIPT  Missing script: dev
> ```
> 
> Should `dev` be replaced with `build` or some other command? Note that `pnpm run dev` _does_ exist in `website/package.json` - just not `package.json`.
> 
> Also, I don't think it's clear from the docs whether that command should be run in the root directory or `packages/`. It'd be nice to clarify.
> 
> Great contributing guide otherwise btw! 🙌

😁 😁 , thanks @JoshuaKGoldberg!

--- a/.github/CONTRIBUTING.md
+++ b/.github/CONTRIBUTING.md
@@ -226,12 +226,6 @@ git checkout -b <your-branch-name>
 
 The project is a mono repo containing the codes for the **website**, the **core Million.js package**, the **playground**, and the **tests**.
 
-To run the project locally, you need to run the development server:
-
-```bash
-pnpm run dev
-```
-
 If you want to run the website locally, you need to `cd` into the website folder before running the development server:
 
 ```bash

